image_provisioner/inventory/inventory

```diff
@@ -1,36 +0,0 @@
-[all:vars]
-#ansible_ssh_private_key_file = $HOME/.ssh/id_rsa_perf
-ansible_ssh_private_key_file = /root/.ssh/id_rsa_perf
-base_image_url=http://download.url/rhel-guest-image-7.2-20160302.0.x86_64.qcow2
-base_image_filename=rhel-guest-image-7.2-20160302.0.x86_64.qcow2
-image_tag=2016-07-27
-# amount of GB to extend root partition by.  default size is 10G so extend that by an additional # gb:
-extend_disk_gb=15
-config_type=ose
-graphite_host=
-graphite_prefix=ose
-collectd_interval=10
-collectd_status=started
-# set openshift_version to the rpm version, sometimes with a trailing -N
-openshift_version=3.3.0.11-1
-# vda1 for openstack. Only necessary for image resize for RHEL cloud image.
-#root_partition=/dev/vda1
-## vda2 for openstack sdb for ec2
-#docker_storage_device=/dev/vda2
-docker_storage_vgname=docker_vg
-docker_login=no
-github_token=
-config_repo=svt-private
-aos_ansible_path=/root/aos-ansible
-aos_ansible_inventory=/root/aos-ansible/inventory/hosts   
-puddle_url=
-# this is the version of the images.  never has the trailing -N like the RPMs might
-image_version=v3.3.0.11
-qe_repo_image_type=ose
-rhel_repo_hostname=
-pbench_internal_hostname=
-pbench_alternate_hostname=
-root_password=
-
-[all]
-# 127.0.0.1
```

Comment:
We shouldn't delete this file entirely since it is a good OpenStack example.   I suggest having inventory.OS and inventory.AWS.   We can add inventory.GCE etc later.